tests/fmt.rs

```diff
@@ -1131,6 +1131,12 @@ mod escapes {
             );
         });
     }
+
+    #[test]
+    fn entity_escape_is_not_code_block_indent() {
```

Comment:
Could you also add a test that tests for bulleted lists? The code seems to specifically handle them, but it's not visible here.